### ISSUE TITLE
How to show uploading & downloading progress for Images,Videos ?

### ISSUE THREAD

**Issue by satnamvision** [stfalcon-studio/ChatKit#109]
how to show upload download progress or progress bar for the images,videos & other custom content types. **I am using default implementation of messageAdapter with some custom content type using messageHolder**. **So my problem is show progress bar while a video/image file is getting upload on the server**. Is there any that can help me with this or **do i need to create custom Message Adapter**?

**Comment by ChiHwe** [comment]
hi, you found way to upload video ? if yes, can show me how to do it ?

**Comment by tusharuit25** [comment]
I am struggling with same problem

**Comment by satnamvision** [comment]
I don't think so. I am using firebase as backend. so I was purposing to change the sequences of operations for uploading & downloading file. I haven't implemented it yet but that was the only way I have found.
